server/src/types.ts

```diff
@@ -0,0 +1,26 @@
+
+export type OMTModule = {
+    name: string,
+    uri: string,
+}
+
+export type CheckFileResult = {
+    path: string,
+} & CheckTextResult
+
+export type CheckTextResult = {
+    module?: {
+        name: string,
+    },
+}
+
+export function isDeclaredImportLinkData(data: unknown): data is DeclaredImportLinkData {
+    const linkData = (<DeclaredImportLinkData>data);
```

Comment:
heheh, so both are 'the typescript way'. I just encountered the `<type>value` syntax recently for casting and it felt more comfortable for me so I used it here. They work exactly the same but `as` was introduced for JSX stuff. 
I don't really mind either way so I'll make it `as` since that is newer.